src/services/task.service.ts

```diff
@@ -93,11 +67,11 @@ export class TaskService {
 
     public async validateTaskData(currentUser: User, taskId: string, taskData: any) {
 
-        if(!currentUser.roles.includes("ADMIN") && !currentUser.roles.includes("MANAGER")) {
+        if(!currentUser.roles.includes(Role.admin) && !currentUser.roles.includes(Role.manager)) {
```

Comment:
Role.admin looks kind of weird, is there any way to maybe use an enum for these places where you use Role?

Could be Role.ADMIN where ADMIN corresponds to 'admin' or 'ADMIN' or however you want

Reply:
Ah I see, seems like you're already using enums, standard way to write enums would be all uppercase, it looks cleaner and shows that it's an enum much better, so something like Role.ADMIN would look better than Role.admin, but that's just my opinion, same comment about the FE enum, so I won't leave a comment there